Cargo.toml

```
@@ -29,6 +29,7 @@ categories = [
 
 [dependencies]
 paste = "1.0.2"
+unsize = "1"
```

Also, by the way, we could start with the dependency being optional:

```suggestion
unsize = { version = "1", optional = true }
```

and then add `#[cfg(feature = "unsize")]` on the two places that'd need it.

  - and, ideally, also add testing that featuring onto the feature matrix, as done in https://github.com/danielhenrymantilla/stackbox.rs/blob/20406f9dabd65ec213dd6724af86c335584a2b0d/.github/workflows/future-proof.yml#L28

    ```diff
      - name: Cargo test
        uses: actions-rs/cargo@v1
        with:
          command: test
          args: |
    -       --no-default-features ${{ matrix.feature--alloc }}
    +       --no-default-features ${{ matrix.feature--alloc }} ${{ matrix.feature--unsize }}

---

(🤔 I seem to have a typo on my CI files: I sometime name that matrix config `feature--alloc` and sometimes `feature--alloc-or-std` 😅 🤦 Just ignore that and I'll fix it on my own end)